lib/jekyll/gitlab/letsencrypt/configuration.rb

```diff
@@ -40,7 +40,7 @@ def layout
           end
 
           def personal_access_token
-            jekyll_config['personal_access_token']
+            ENV['GITLAB_TOKEN'] || jekyll_config['personal_access_token']
```

Comment:
also maybe re-arrange so token passed via config is taken always ? In this situation someone could see this as a bug, a lot of other apps could use `GITLAB_TOKEN` too and it could not be user intent to use it in jekyll gitlab letsencrypt.